AISKU/src/Initialization.ts

```diff
@@ -179,6 +179,11 @@ export class Initialization implements IApplicationInsights {
                 const ingest = cs.ingestionendpoint;
                 config.endpointUrl = ingest ? `${ingest}/v2/track` : config.endpointUrl; // only add /v2/track when from connectionstring
                 config.instrumentationKey = cs.instrumentationkey || config.instrumentationKey;
+            } else {
+                var _console = typeof console !== strUndefined ? console : null;
+                if (_console && _console.warn) {
+                    _console.warn("Instrumentation key based ingestion will be no longer supported, please use connection string instead.");
```

Comment:
Make sure we either include a aka link (go.microsoft.com) pointing to the deprecation notice for ikeys.  I think this is one potential document we can refer to: https://docs.microsoft.com/en-us/azure/azure-monitor/app/migrate-from-instrumentation-keys-to-connection-strings

In any case " no longer supported" is not accurate.  We stop supporting it sometime in future.